js/packages/store/providerEffects.ts

```diff
@@ -245,11 +254,27 @@ export const openingDaemon = async (
 
 	accountService
 		.getGRPCListenerAddrs({})
-		.then(() => {
+		.then(async rep => {
```

Comment:
Why an async func here?

Reply:
Yep i'll refacto this function

Reply:
In the same case i can refacto some other 👍 like in `openingClients` in the same file